web_src/js/index.js

```diff
@@ -304,11 +305,27 @@ function initImagePaste(target) {
   });
 }
 
+function initSimpleMDEImagePaste(simplemde, files) {
+  simplemde.codemirror.on('paste', (_, event) => {
+    retrieveImageFromClipboardAsBlob(event, (img) => {
+      const name = img.name.substr(0, img.name.lastIndexOf('.'));
+      uploadFile(img, (res) => {
+        const data = JSON.parse(res);
+        const pos = simplemde.codemirror.getCursor();
+        simplemde.codemirror.replaceRange(`![${name}](${suburl}/attachments/${data.uuid})`, pos);
+        const input = $(`<input id="${data.uuid}" name="files" type="hidden">`).val(data.uuid);
+        files.append(input);
+      });
+    });
+  });
+}
+
 function initCommentForm() {
   if ($('.comment.form').length === 0) {
     return;
   }
 
+  setCommentSimpleMDE($('.comment.form textarea'));
   initBranchSelector();
   initCommentPreviewTab($('.comment.form'));
   initImagePaste($('.comment.form textarea'));
```

Comment:
Isn't `initImagePaste` has done that?

Reply:
Sorry, I didn't understand what you mean. Do you mean why I create a new function `initSimpleMDEImagePaste` to fix the image paste problem? 🤔

Reply:
@lunny, if you are asking why there is a new `initSimpleMDEImagePaste`, the reasons are:

1. The `initImagePaste` always attached the uploaded file input `$('.files')`, which is not the correct DOM element for comment. So I need a new function to allow caller providing correct DOM element to attach file.

2. When I trying to re-use `initImagePaste` for comment editor, it's weird that the `CodeMirror` input-field fails to insert image (`![img](http://url)`.  Therefore I have to insert the image markdown with `simplemde.codemirror.replaceRange`.